server/src/resolvers/user/logout.ts

```diff
@@ -1,10 +0,0 @@
-import { Context } from '../../types'
-
-export async function logoutUser ({ req }: Context) : Promise<void> {
-  console.log(req.session)
-  await new Promise<void>((resolve) => {
-    req.session.destroy(resolve)
-  })
-
-  console.log(req.session)
-}
```

Comment:
We don't need that anymore?

Reply:
I moved it into the resolver itself, same as self.ts